multidc/src/main/resources/application.conf

```diff
@@ -54,7 +54,11 @@ akka {
       "akka.tcp://MultiDcSystem@127.0.0.1:2552"
     ]
 
-    multi-data-center.self-data-center = "eu-west"
+    multi-data-center {
+      self-data-center = "eu-west"
+      
+      speculative-replication.enabled = off
```

Comment:
@chbatey here is a PR for trying the speculative replication, set this to `on`. It's supposed to reduce number of requests to C* since the messages are assumed to arrive earlier and the stage can be fast forwarded.